### PR TITLE
T36340 Allow iteration of procedure phases

### PR DESCRIPTION
**Pull request by martendemos**
**Phab-Ticket:** [T36340](https://yaits.demos-deutschland.de/T36340)
**Ticket:** [DPP-3274](https://demoseurope.youtrack.cloud/issue/DPP-3274/5-2-Part-1-Durchgangsnummer-zu-Verfahrensschritten-hinzufugen)

Allow count the iterations of a procedure phase.

### How to review/test
The most of the work will be done in the [preparation PR](#2826), in which the phase of procedure is extracted to its own entity.
In this one, only the additional field is added and will be handled.

### Linked PRs (optional)
#2826
#2834
https://github.com/demos-europe/demosplan-addon/pull/98

### Tasks (optional)
- [x] adjust curds

### PR Checklist
- [x] Tests updated/created
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly
- [ ] merge https://github.com/demos-europe/demosplan-addon/pull/98 immediately after this PR is merged

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php [hunk 1]
```diff
@@ -22,7 +22,8 @@ class Version20240313120508 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return 'refs T36340: Extract phase of an procedure into own entity. Step2: Copy current data into new entity.';
```
Comment (breierDemos) [inline demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php:25]: I think these changes are not intended for this pr?

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php [hunk 2]
```diff
@@ -46,6 +47,13 @@ public function up(Schema $schema): void
     public function down(Schema $schema): void
     {
         $this->abortIfNotMysql();
+
```
Comment (breierDemos) [inline demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php:50]: I think these changes are not intended for this pr? Does the same as the up migration :thinking:

**Review by breierDemos — CHANGES_REQUESTED** [review]
Code Review only, not sure about twig changes.

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php (hunk 1 above)
Comment (martendemos) [reply demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php:25]: yes ok, but doesn't matter its only an formatting improvement, which also can be done here.

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php (hunk 2 above)
Comment (martendemos) [reply demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php]: solved by merge target branch

**Pull request merged**